docs/bundles/helm.md

```diff
@@ -0,0 +1,50 @@
+# Helm Bundle Spec
+
+## Overview
+
+This document is meant to define the helm bundle format as a reference for those publishing helm bundles for use with
+RukPak. A bundle is a collection of Kubernetes resources that are packaged together for the purposes of installing onto
+a Kubernetes cluster. A bundle can be unpacked onto a running cluster, where controllers can then create the underlying
+content embedded in the bundle. The bundle can be used as the underlying `spec.source` for
+a [Bundle](https://github.com/operator-framework/rukpak#bundle) resource.
```

Comment:
We repeat this paragraph for every bundle document, perhaps we should update it to just point to the `bundle` section of the root readme instead and say something like:
```suggestion
This document is meant to define the helm bundle format as a reference for those publishing helm bundles for use with RukPak. For more information on the concept of a bundle, click [here](https://github.com/operator-framework/rukpak#bundle).
```
Applies to all subsequently similar entries.

Reply:
I'll update.  Thanks!